Sources/Testing/Test.swift

```diff
@@ -89,7 +89,6 @@ public struct Test: Sendable {
   }
 
   /// Whether or not this test is parameterized.
-  @_spi(ExperimentalParameterizedTesting)
   public var isParameterized: Bool {
     guard let parameterCount = parameters?.count else {
```

Comment:
While you're in here, you could change this to `isEmpty == false`?

Reply:
Done